src/neural/network_legacy.cc

```diff
@@ -21,6 +21,7 @@
 #include <cmath>
 #include <algorithm>
 #include "utils/weights_adapter.h"
+#include "utils/logging.h"
```

Comment:
Not in alphabetical order.

Reply:
Actually this is a leftover from a debugging session, should be removed.